### DISCUSSION TITLE
Container queries for tables

### DISCUSSION THREAD

**Discussion by matusko89** [primefaces/community#1125] [PrimeNG] [answered]
Hey, current table has an option to switch to stack layout based on breakpoint value. This is currently iplemented via meadia queries. I thing the better option would be to implement this feature via Container queries. This would be much moe usefull if you have and dynamic component that can be added anywhere on page.

**Comment by mehmetcetin01140** [comment, marked as answer]
Thanks for the feedback. We have passthrough and unstyled mode in our roadmap. We can do it after we complete them.
